settings.gradle.kts

```diff
@@ -14,8 +14,8 @@ pluginManagement {
 }
 
 val javaVersion = JavaVersion.current()
-require(javaVersion.isJava12Compatible) {
-	"The JUnit 5 build requires Java 12 or higher. Currently executing with Java ${javaVersion.majorVersion}."
+require(javaVersion.isJava13Compatible) {
+	"The JUnit 5 build requires Java 13 or higher. Currently executing with Java ${javaVersion.majorVersion}."
```

Comment:
Oh, man. I just installed Java 12 yesterday to build JUnit, then you upgrade it to 13. That is way too fast :)

Reply:
"12 is dead, Jim."

Just waiting for Gradle 6.0 to be released...